welcome-to-sourcery.py

```diff
@@ -7,10 +7,10 @@
 
 # Let's start looking at how you can get a code review from Sourcery.
 
-# The `Review` tab allows you to get a code review straight away in your IDE.
-# You can choose between reviewing your current uncommitted changes and reviewing
-# the changes between your current branch and the default branch.
-# (Note that this will only work on code that is under Git version control).
+# The `Review` tab allows you to get a code review straight away in your IDE
+# - you can always get a review of the current file
+# - if in Git you can review your current set of uncommitted changes, 
```

Comment:
nit: I would capitalize the start of each of the bullet points, but that's fully a stylistic pref